tests/cli/yubihsm/keys/list.rs

```diff
@@ -10,6 +10,15 @@ fn keys_command_test() {
     #[cfg(feature = "yubihsm-mock")]
     args.extend_from_slice(&["-c", super::KMS_CONFIG_PATH]);
 
-    // TODO: parse results
-    cli::run_successfully(args.as_slice());
+    let out = cli::run_successfully(args.as_slice());
+
+    assert_eq!(true, out.status.success());
+    assert_eq!(true, out.stdout.is_empty());
+    assert_eq!(
+        true,
+        String::from_utf8(out.stderr)
+            .unwrap()
+            .trim()
+            .starts_with("error: no keys in this YubiHSM")
```

Comment:
We can probably add more realistic integration tests too. 
sth along the lines of: 
list -> generate -> list -> import -> list

Reply:
The main problem with that sort of thing right now is the MockHSM does not persist its state between tests. I could look into some way to do that though (if nothing else just making its internal state `Serialize`able and dumping it to a JSON file or something)

Reply:
I think that would be neat. But I also think it's probably not a high priority I guess.